packages/govuk-frontend/src/govuk/components/character-count/_index.scss

```diff
@@ -14,7 +14,7 @@
   }
 
   .govuk-character-count__message {
-    @include govuk-font($size: false, $tabular: true);
+    @include govuk-typography-tabular-numbers;
```

Comment:
Quite like your `govuk-font-variant-tabular-nums` suggestion

If we use `font-variant-numeric` we might want `-numeric` too:

```scss
@include govuk-font-variant-numeric-tabular-nums;
@include govuk-font-variant-numeric-oldstyle-nums; // Maybe one day
```

But made me realise that `font-variant` already works in IE11

```scss
@include govuk-font-variant-tabular-nums;
@include govuk-font-variant-oldstyle-nums; // Maybe one day
```